update.cf.in

```diff
@@ -18,7 +18,7 @@ body common control
                           @(update_def.bundlesequence_end), # Define control_common_update_bundlesequnce_end via augments
       };
 
-      version => "update.cf $(update_def.current_version)";
+      version => "update.cf @VERSION@";
```

Comment:
why have this here? I thought we might get rid of `@VERSION@`?
Why not stick with update_def.current_version even though now that is derived from a few promises: create the cf_version.txt file, read it...

Reply:
This is the policy version, which I still think we should seed (for knowledge). Previously it got here indirectly. So I moved it to direct and then changed current_version to align with the hub binary. Don't even get me started on bad variable names.